fiks-meldingsformidler/src/main/java/no/difi/meldingsutveksling/ks/svarut/SvarUtConnectionCheck.java

```diff
@@ -0,0 +1,21 @@
+package no.difi.meldingsutveksling.ks.svarut;
+
+import lombok.RequiredArgsConstructor;
+import no.difi.meldingsutveksling.nextmove.NextMoveRuntimeException;
+
+import javax.annotation.PostConstruct;
+
+@RequiredArgsConstructor
+public class SvarUtConnectionCheck {
+
+    private final SvarUtService svarUtService;
+
+    @PostConstruct
+    public void checkTheConnection() {
+        try {
+            svarUtService.getMessageReceipt("1");
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)